R/metab_simple.R

```diff
@@ -24,7 +24,9 @@
 #'   
 #' @author Alison Appling, Jordan Read; modeled on lakeMetabolizer
 #' @examples
-#'  metab_simple(data=data.frame(date.time=rep(as.Date("2015-04-15"), 24), DO.obs=1:24, DO.sat=rep(14, 24), PAR=sin((1:24)*pi/24)^8, temp.water=15))
+#'  metab_simple(data=data.frame(
```

Comment:
How about a separate function that does all the data preparation for you?
Then we can also (later) create a high-level function that both prepares
and models the data, but our core modeling functions will remain relatively
short and focused on a single activity (fitting a model).

On Thu, Apr 23, 2015 at 7:54 AM, Jordan S Read <notifications@github.com>
wrote:

> In R/metab_simple.R
> <https://github.com/USGS-R/streamMetabolizer/pull/11#discussion_r28959102>
> :
>
> > @@ -24,7 +24,9 @@
> >  #'
> >  #' @author Alison Appling, Jordan Read; modeled on lakeMetabolizer
> >  #' @examples
> > -#'  metab_simple(data=data.frame(date.time=rep(as.Date("2015-04-15"), 24), DO.obs=1:24, DO.sat=rep(14, 24), PAR=sin((1:24)*pi/24)^8, temp.water=15))
> > +#'  metab_simple(data=data.frame(
>
> Ok, but a lot of use cases assume salinity~0, and use altitude for baro,
> so since wtr.temp and do.obs are already there, it doesn't seem onerous to
> support
>
> metab_simple(data = data.frame(date.time=rep(as.Date("2015-04-15"), 24), DO.obs=1:24, temp.water=15), altitude=424)# as input to
> metab_simple(data, ...)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/USGS-R/streamMetabolizer/pull/11/files#r28959102>.
>



-- 
Alison Appling
Postdoctoral Fellow, USGS Powell Center
USGS Center for Integrated Data Analytics and University of Wisconsin -
Madison
Phone: (608) 821-3927